spec/command_spec.rb

```diff
@@ -164,6 +164,33 @@
         end
         @command.run '--interval', '15'
       end
+
+      describe 'given a global option' do
```

Comment:
This looks great! Could you also add a "integration" type spec to `runner_spec.rb`? I took a quick stab here: https://github.com/commander-rb/commander/commit/0d8cfe47a6e6e4aa463e610bbfd4e044673d46b8 

Feel free to pull in that commit or rewrite as you see fit. Thanks!

Reply:
Awesome! I've merged that in.